lib/bootstrap.js

```diff
@@ -0,0 +1,65 @@
+var spawn = require('child_process').spawn;
+var path = require('path');
+var sleep = require('sleep').sleep;
+
+function Bootstrap(codecept, callback) {
+
+  var phantomjs, seleniumPid, callbackCalled = false;
+
+  (function () {
+
+    var protractor = codecept.config.helpers.Protractor;
+    if (protractor && protractor.browser) {
+      if (/phantom/.test(protractor.browser)) {
+        var phantomPath = require('phantomjs').path || '/usr/local/bin/phantomjs';
+        phantomjs = spawn(phantomPath,['--webdriver=4444']);
+
+        phantomjs.stdout.on('data', function (data) {
+          if (!callbackCalled) {
+            callback();
+            callbackCalled = true;
+          }
+        });
+
+        process.on('exit', function () {
+          phantomjs.stdin.pause();
+          phantomjs.kill();
+        });
+
+      }else {
+
+        var binPath = path.join(path.dirname(require.resolve('protractor')),'..','..','.bin');
```

Comment:
Which line? The `require.resolve()` call to mount the path? So, this is to get the binary path of `webdriver-manager` package. It is a dependency of `Protractor`